nh_observations/notifications.py

```diff
@@ -129,6 +129,11 @@ def complete(self, cr, uid, activity_id, context=None):
         }
     ]
 
+    def set_form_description_frequencies(self, available_frequencies):
```

Comment:
How does this play with https://github.com/NeovaHealth/openeobs/blob/develop/nh_ews/notifications.py#L139 ?

Setting the observation frequencies on the nh_observation object will mean that the change to the form description in nh_ews will most likely override this dictionary anyways